app/views/calculators/calculator.html.slim

```diff
@@ -11,10 +11,11 @@
             = form.text_field :child_birthday, class:"form-input", id: "date", onfocus:"(this.type='date')", placeholder:"Child`s birthday", onblur:"if(!this.value) this.type='text'", required: true
             = form.text_field :baby_weight, placeholder:"Baby weight", class:"form-input", required: true
             = form.text_field :diapers_per_day, placeholder: "Diapers per day", class:"form-input", required: true
+          - if user_signed_in?
```

Comment:
I recommend to move this logic to `helpers`.
In the view we will call helper method 
`= render_email_receiver_checkbox`
And all logic will move to rails helpers
```ruby
def render_email_receiver_checkbox
  return unless user_signed_in?
    .....
end
```
Some examples https://stackoverflow.com/questions/4576932/ruby-on-rails-how-to-render-a-string-as-html

As a result we need test cases to cover signed in and not signed in users.